ProcessMaker/Repositories/CaseRepository.php

```diff
@@ -110,6 +102,10 @@ public function update(ExecutionInstanceInterface $instance, TokenInterface $tok
      */
     public function updateStatus(ExecutionInstanceInterface $instance): void
     {
+        if (!is_null($instance->parent_request_id)) {
```

Comment:
Add a code comment to exaplin why this validation is done

Reply:
Comment was added